src/pro/users.md

```diff
@@ -0,0 +1,21 @@
+# Users and Permission
+
+The Pro version of Geo Engine includes a user management system.
+Users can either be _anonymous_ or _registered_.
+On the first startup, an admin user will be created.
+
+Geo Engine has a Role Based Access Control (RBAC) system.
+Users can have different roles and permissions on resources are granted to these roles.
+By default, they have a unique role for themselves and either the role `anonymous` or `registered`.
+The admin user has the role `admin`.
+
```

Comment:
can we add an example or is that to complex?

Reply:
what do you have in mind? Give an example for a resource and a role and some permissions and explain who has access and why? I can do that if you want.

Reply:
Exactly. Something like this so the reader can "see" what a permission is and what it does.

Reply:
147c86aa092324758bce9c9a96be08122a4bbb18